src/pages/dolacz-do-nas/index.tsx

```diff
@@ -37,36 +38,38 @@ const JoinUsPage = () => (
       description="Jeśli chcesz razem z nami tworzyć projekty społeczne, badawcze, budować rzeczywiste obiekty i mieć realny wpływ na życie w mieście!"
       image={{ src: '/images/dolacz-do-nas/ekipa-hsm-600x500.png', alt: 'Ekipa HŚM' }}
     >
-      <LinkButton href="/" className="relative my-8 bg-neutral/10 text-neutral/90">
-        <span className="z-10">Dołącz do nas</span>
-        <div className="absolute top-2 left-2 hidden h-full w-full rounded-20 border-2 border-solid border-neutral/10 md:block" />
+      <LinkButton href="/" className="relative mt-8 w-[147px] bg-primary/20 py-2 px-6 text-neutral/90 md:flex">
+        <Paragraph className="z-10">Dołącz do nas</Paragraph>
+        <div className="absolute top-2 left-2 hidden h-full w-full rounded-20 border-2 border-solid border-primary/20 md:block" />
       </LinkButton>
     </TopSection>
 
     <MainSection className="text-2xl">
       <div className="mx-auto max-w-screen-xl px-4 text-center">
-        <h2 className="py-16 text-center text-4xl">Dlaczego warto?</h2>
-        <div className="grid-flow-col md:grid">
-          <div className="mb-14 md:mx-6">
+        <h2 className="py-16 text-center">Dlaczego warto?</h2>
+        <div className="mb-36 grid-flow-col md:grid">
+          <div className="flex flex-col gap-10 md:mx-6">
             <Image
               src={GroupOfPeopleIcon}
-              height={40}
-              width={50}
+              height={64}
+              width={64}
               alt="group of people icon"
-              className="mx-auto mb-8 md:mb-4"
+              className="mx-auto h-16 md:mb-4"
             />
-            <p>Poznasz wiele aktywnych i twórczych osób, z którymi możesz rozwijać swoje umiejętności i pasje</p>
+            <p className="text-2xl font-normal">
+              Poznasz wiele aktywnych i twórczych osób, z którymi możesz rozwijać swoje umiejętności i pasje
+            </p>
           </div>
-          <div className="mb-14 md:mx-6">
-            <Image src={MagnifierIcon} height={45} width={45} alt="magnifier icon" className="mx-auto mb-8 md:mb-4" />
-            <p>
+          <div className=" flex  flex-col gap-10 md:mx-6">
```

Comment:
Missing a little bit of margin between elements :)

Reply:
Repaired. Please check if this is okay now